src/components/organisms/Navbar/Navbar/__snapshots__/Navbar.test.tsx.snap

```diff
@@ -5,6 +5,22 @@ exports[`<Navbar /> should render component with all the props 1`] = `
   height: 80px;
 }
 
+@media (min-width:576px) {
+
+}
+
+@media (min-width:768px) {
+
+}
+
+@media (min-width:992px) {
+
+}
+
+@media (min-width:1300px) {
+
+}
+
```

Comment:
that's a bit strange

Reply:
yes... provably a bug in jest-styled-components... It's a bit flacky with v5 of styled-components. I read they change many things internally